include/bluetooth/services/fast_pair.h

```diff
@@ -23,30 +23,33 @@ extern "C" {
 /** Value that denotes unknown battery level (see @ref bt_fast_pair_battery). */
 #define BT_FAST_PAIR_BATTERY_LEVEL_UNKNOWN	0x7f
 
-/** @brief Fast Pair advertising mode. Used to generate advertising packet.
- *
- * According to Fast Pair specification, when no Account Key has been saved on the device both Fast
- * Pair not discoverable advertising modes result in the same advertising data.
- */
+/** @brief Fast Pair advertising mode. Used to generate advertising packet. */
 enum bt_fast_pair_adv_mode {
 	/** Fast Pair discoverable advertising. */
-	BT_FAST_PAIR_ADV_MODE_DISCOVERABLE,
+	BT_FAST_PAIR_ADV_MODE_DISC,
 
-	/** Fast Pair not discoverable advertising, show UI indication. */
-	BT_FAST_PAIR_ADV_MODE_NOT_DISCOVERABLE_SHOW_UI_IND,
-
-	/** Fast Pair not discoverable advertising, hide UI indication. */
-	BT_FAST_PAIR_ADV_MODE_NOT_DISCOVERABLE_HIDE_UI_IND,
+	/** Fast Pair not discoverable advertising. */
+	BT_FAST_PAIR_ADV_MODE_NOT_DISC,
 
 	/** Number of Fast Pair advertising modes. */
 	BT_FAST_PAIR_ADV_MODE_COUNT
 };
 
+/** @brief Fast Pair not discoverable advertising type. Used to generate advertising packet. */
+enum bt_fast_pair_not_disc_adv_type {
+	/** Show UI indication. */
+	BT_FAST_PAIR_NOT_DISC_ADV_TYPE_SHOW_UI_IND,
+
+	/** Hide UI indication. */
+	BT_FAST_PAIR_NOT_DISC_ADV_TYPE_HIDE_UI_IND
+}
+
 /** @brief Fast Pair advertising battery mode. Used to generate advertising packet.
  *
  * Battery data can be included in advertising packet only if the Fast Pair Provider is in Fast Pair
- * not discoverable advertising mode. To prevent tracking, the Fast Pair Provider should not include
- * battery data in the advertising packet all the time.
+ * not discoverable advertising mode and is in possesion of at least one Account Key. To prevent
```

Comment:
We asked Google this question by mail thread. Specification itself is unclear because here https://developers.google.com/nearby/fast-pair/specifications/service/provider#advertising_payload_fast_pair_account_data it says that Account Key Data should be 0x00 when Account Key List is empty and here https://developers.google.com/nearby/fast-pair/specifications/extensions/batterynotification#BatteryNotification it explains how to change Account Key Data when Battery info is included.

Reply:
I see, thanks for explanation